cmd/generate.go

```diff
@@ -103,6 +104,49 @@ func genProcessCluster(cmd *cobra.Command, clusterName string, p *ants.Pool) {
 	}
 	sort.Strings(compList) // process components in sorted order
 
+	// List of cluster components in kr8
+	var clComps []string
+	for c, _ := range clusterComponents {
+		clComps = append(clComps, c)
+	}
+	sort.Strings(clComps)
+
+	// List of generated components for cluster
+	d, err := os.Open(clusterDir)
+	if err != nil {
+		log.Fatal().Err(err).Msg("")
+	}
+	defer d.Close()
+	compDirs, err := d.Readdirnames(-1)
+	if err != nil {
+		log.Fatal().Err(err).Msg("")
+	}
+	d.Close()
```

Comment:
This is already deferred.

Reply:
ah I see, seems it doesn't need this after the refactor. removing

Reply:
gone! :)